src/coreclr/src/jit/inline.h

```diff
@@ -541,9 +541,11 @@ struct InlineCandidateInfo : public GuardedDevirtualizationCandidateInfo
 
 struct InlArgInfo
 {
-    GenTree* argNode;                     // caller node for this argument
-    GenTree* argBashTmpNode;              // tmp node created, if it may be replaced with actual arg
-    unsigned argTmpNum;                   // the argument tmp number
+    GenTree*         argNode;             // caller node for this argument
+    GenTree*         argBashTmpNode;      // tmp node created, if it may be replaced with actual arg
+    unsigned         argTmpNum;           // the argument tmp number
+    unsigned __int64 bbFlags;             // basic block flags that need to be added when replacing GT_RET_EXPR
```

Comment:
Nit: I'd place this field first to minimize the amount of padding we might end up with here.

Reply:
Sure, done.